tests/test_filters.py

```diff
@@ -175,6 +176,16 @@ def test_row_col_medians_with_mask(
     np.testing.assert_array_equal(medians["cols"], image_random_col_medians_masked)
 
 
+def test_non_square_img(test_filters_random: Filters):
+    test_filters_random.images["pixels"]=test_filters_random.images["pixels"][:,0:512]
+    test_filters_random.images["zero_averaged_background"] = test_filters_random.average_background(
+        image=test_filters_random.images["pixels"], mask=None
+    )
+    assert isinstance(test_filters_random.images["zero_averaged_background"], np.ndarray)
+    assert test_filters_random.images["zero_averaged_background"].shape==(1024, 512)
+    assert np.sum(test_filters_random.images["zero_averaged_background"]) == 44426.48188033322
```

Comment:
Very minor and pretty much inconsequential, but arrays have the `.sum()` method that you can apply to them so you could here do the following which would be consistent with the other tests...

```suggestion
    assert test_filters_random.images["zero_averaged_background"].sum() == 44426.48188033322
```

But its incredibly pedantic of my to highlight this and doesn't really need changing.

Reply:
Ah I didn't realise this. I've made the appropriate changes